### PR TITLE
[NETBEANS-54] excluding specific files, e.g., licenses via Rat

### PR DESCRIPTION
**Pull request by geertjanw**

### CODE REVIEW

File: nbbuild/build.xml [hunk 1]
```diff
@@ -2024,8 +2027,11 @@ It is possible to use -Ddebug.port=3234 -Ddebug.pause=y to start the system in d
             <exclude name="editor.plain/src/org/netbeans/modules/editor/plain/resources/PlainTextExample" /> <!--license would be visible to users in the Fonts/Colors settings-->
             <exclude name="editor.settings.storage/test/unit/src/org/netbeans/modules/editor/settings/storage/compatibility/p1/**" /> <!--test data-->
             <exclude name="extbrowser/test/unit/src/org/netbeans/modules/extbrowser/data/mac_defaults_*" /> <!--test data-->
+            <exclude name="form/src/org/netbeans/modules/form/resources/templates/**" /> <!--template files-->
             <exclude name="hibernate/src/org/netbeans/modules/hibernate/resources/Hibernate*.xml" /> <!--template files-->
             <exclude name="hibernate/src/org/netbeans/modules/hibernate/resources/HibernateUtil.javax" /> <!--template file-->
+            <exclude name="html/src/org/netbeans/modules/html/templates/**" /> <!--template files-->
```
Comment (matthiasblaesing) [inline nbbuild/build.xml]: This should not be here. The license header there is CDDL-GPL-2-CP and needs to be adjusted. I fixed that in #272.

**Comment by geertjanw** [comment]
So, should I simply close this one, is everything handled in #272?

**Comment by matthiasblaesing** [comment]
No, #272 makes the exclude you introduced in 2033 obsolete, so that should be removed, The `VERSIONS.txt` excludes can be replaced with a single wildcard exclude as shown above.

**Comment by matthiasblaesing** [comment]
@geertjanw I extended your work by adding the ALv2 header to all VERSION.txt files. You established this in a prior fix and I think it is a good idea. Please see here:

https://github.com/matthiasblaesing/incubator-netbeans/tree/pr271-followup

You can full that into your tree as:
```
git pull https://github.com/matthiasblaesing/incubator-netbeans.git pr271-followup
```

**Comment by geertjanw** [comment]
OK, I'll use that to pull this into my tree, but does anything from here need to be pushed to Apache NetBeans Git? Sorry, it's getting too complex for me in this PR, if there's a way you can commit this, that would be great.

**Comment by geertjanw** [comment]
Thank you! :-)

**Pull request merged**